src/node_url.cc

```diff
@@ -2079,6 +2081,69 @@ static void DomainToUnicode(const FunctionCallbackInfo<Value>& args) {
                           v8::NewStringType::kNormal).ToLocalChecked());
 }
 
+std::string URL::ToFilePath() {
+  if (context_.scheme != "file:") {
+    return "";
+  }
+
```

Comment:
Okay! So for posix it would resolve to `"/dev"` and for win32 it would resolve to `""`?